### ISSUE TITLE
Can the developer(s) perhaps give us weekly updates on the status so we can have a rough idea when the code might be published officially?

### ISSUE THREAD

**Issue by Collin-Budrick** [HumanAIGC/AnimateAnyone#48]
I recommend creating a Discord server for the project and setting up an announcement channel like a lot of AI projects do.

**Comment by ReEnMikki** [comment]
The Issues section of this repo is already such a cesspool, I can't imagine the hellhole that is called the Discord server for this project 💀💀💀

Also looks like this repo has reached its plateau. The rate of it gaining stars with zero code has practically became minimal now. 10k+ ⭐ with no code is always funny.

**Comment by Collin-Budrick** [comment]
> The Issues section of this repo is already such a cesspool, I can't imagine the hellhole that is called the Discord server for this project 💀💀💀

Yeah, well, the authors can't give a date when they will release the code, and they don't put out updates routinely. I'm beginning to believe the rumors that this was at least partly fake. It shouldn't take this long to polish up the code for ease of use.

There's a number of people on this repo who are already putting the paper into code by themselves. Kinda embarrassing/suspicious at this point that they haven't even released the model at this point.

**Comment by cepera-ang** [comment]
It is not up to authors only to decide to publish corporate intellectual
property. Especially after public stir that happened here. You may be
pretty sure that every corporate function now feels very averse to ever
allowing this code to be published. First, it is clearly seem more valuable
now, given the reaction of disbelief. Second, it is now risky that research
code will be treated with expectations of a polished product. Third, all
the SJW activity around model likeness use.

So, quite foreseeable if this code release will switch from "interns gonna
publish when they feel comfortable" to "Alibaba will publish it when and if
legal and marketing and big bosses feel comfortable and if deemed
necessary".

**Comment by Collin-Budrick** [comment]
@cepera-ang they already committed in their statement where they were with the process... at the very least, they should be transparent about what they're up to through weekly updates.

**Comment by chrisbward** [comment]
Stop being "impatient and shallow-minded"? You turned quickly. Call it how I see it.

**Comment by Collin-Budrick** [comment]
> Stop being "impatient and shallow-minded"? You turned quickly. Call it how I see it.

Context is key... im not accusing them of fake work, I'm merely asking for updates on the progress.

**Comment by lucasjinreal** [comment]
Please do not hope a none existed project opensource, this is just a PR. Don't be too serious.

**Comment by Collin-Budrick** [comment]
> Please do not hope a none existed project opensource, this is just a PR. Don't be too serious.

Then why the f would they post that update on the read me?

**Comment by lucasjinreal** [comment]
It's Chinese style. Lots of README like opensource out there for Chinese companies.

**Comment by feiyangsuo** [comment]
I'm afraid Alibaba won't make this project open-source even if it's truely performant like the teaser shows. Companies like ByteDance could productize it quick. No reason to help business rivals.

**Comment by continue-revolution** [comment]
Some previous projects from Alibaba have a 2-month gap between release and open-source because they have a long internal review process. Actually a lot of Chinese companies / research labs have. So let's wait patiently.

**Comment by Collin-Budrick** [comment]
@continue-revolution after looking at this authors other projects, there doesn't seem to be any code... just baseless marketing nonsense.

**Comment by continue-revolution** [comment]
Yes this group seems to be releasing nothing. Different Alibaba lab have different style. I guess this one is like, well let’s just create a blank repo and see if it can be hot, if it is then let’s make it a product, otherwise let’s open-source. DragonDiffusion (not from Alibaba) takes 7 months to open-source, so who knows. Well now this project has already become a product in their own TongYi app and no one knows if they gonna open-source

**Comment by ShawnFumo** [comment]
Speaking of diff Alibaba labs, a diff person animation project called CHAMP was just released and already has some code and weights available: https://github.com/fudan-generative-vision/champ
